plugins/thank-after-post-plugin-v8.php

```diff
@@ -0,0 +1,13 @@
+<?php
```

Comment:
no need to store plugin in the same repository. You should download it from the provided github link